.idea/JMRI.iml

```diff
@@ -2,7 +2,7 @@
 <module type="JAVA_MODULE" version="4">
   <component name="FacetManager">
     <facet type="Python" name="Python">
-      <configuration sdkName="" />
+      <configuration sdkName="Python 2.7" />
```

Comment:
Is this change deliberate? I ask since sometimes these IDE file gets updated by misstake.

Reply:
adding this (and not having other developers set it higher) is needed for IntelliJ to interpret the jython). But you are correct that some settings could easily slip if those files are not ignored. Python is relevant to this PR so I thought to share this too.